src/main/java/ua/bank/bankservice/service/impl/RoleServiceImpl.java

```diff
@@ -0,0 +1,24 @@
+package ua.bank.bankservice.service.impl;
+
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+import ua.bank.bankservice.model.Role;
+import ua.bank.bankservice.repository.RoleRepository;
+import ua.bank.bankservice.service.RoleService;
+
+@Service
+@AllArgsConstructor
+public class RoleServiceImpl implements RoleService {
+    private final RoleRepository roleRepository;
+
+    @Override
+    public Role create(Role role) {
+        return roleRepository.save(role);
+    }
+
+    @Override
+    public Role getByName(String name) {
+        return roleRepository.getByName(Role.RoleType.valueOf(name))
+                .orElseThrow(() -> new RuntimeException("Can't get role '" + name + "' from db"));
```

Comment:
You can create your own exception. Smth like `EntityNotFoundException`. In the future, you can catch it in GlobalExceptionHandler